rsk-stackoverflow-stats/index.js

```diff
@@ -0,0 +1,102 @@
+#! /usr/bin/env node
+import axios from 'axios';
+import colors from 'colors';
+
+colors.enable();
+
+// fetches all items returned by the specified API URL.
+// repeats until it responds with `has_more: false`
+async function fetchAllItems(url) {
+  let itemList = [];
+  const fetchSequentially = async (pageNumber = 1) => {
```

Comment:
instead of a default value, better to have a regular param, and then inside the function validate that a number was passed in (throw error if not)